ckanext/switzerland/dcat-ap-switzerland_scheming.json

```diff
@@ -1072,6 +1072,23 @@
         "fr": "Un service de données qui donne accès à cette ressource.",
         "it": "Un servizio dati che dà accesso a questa risorsa."
       }
+    },
+    {
+      "field_name": "temporal_resolution",
+      "display_snippet": "text.html",
+      "validators": "scheming_required ogdch_validate_duration_type",
+      "label": {
+        "en": "Temporal Resolution",
+        "de": "Zeitliche Auflösung",
+        "fr": "Résolution temporelle",
+        "it": "Risoluzione Temporale"
+      },
+      "help_text": {
```

Comment:
`help_text` is only used in form snippets, so if we are not including a form snippet, we don't need this.

Reply:
ok, will remove it